frontend/lib/parse-capi/index.ts

```diff
@@ -345,8 +349,7 @@ export const extractArticleMeta = (data: {}): CAPIType => {
         ),
         pageId: getNonEmptyString(data, 'config.page.pageId'),
         sharingUrls: getSharingUrls(data),
-        pillar:
-            findPillar(getNonEmptyString(data, 'config.page.pillar')) || 'news',
+        pillar: findPillar(getString(data, 'config.page.pillar', '')) || 'news',
```

Comment:
Should ` || 'news'` go here or in `findPillar`?

Reply:
that feels like a much better approach :)

Reply:
In fact, findPillar will always return a `Pillar | undefined` so keeping  things as they are probably makes sense here after all.